content/SCALE/SCALEUIReference/Apps/LaunchDockerImageScreens.md

```diff
@@ -0,0 +1,197 @@
+| **Environment Variable Value** | Enter the value for the variable specified in **Environment Variable Name**. For example, for Pi-Hole timezone variable, enter *AmericaNewYork*. |
+{{< /expand >}}
+
+## Networking
+The **Networking** settings specify network policy, addresses, and DNS services if the container needs special networking configuration.
+{{< expand "Click Here for More Information" "v" >}}
+See the [Docker documentation](https://docs.docker.com/network/host/) for more details on host networking. 
+Users can create additional network interfaces for the container if needed or give static IP addresses and routes to new interface. 
+By default, containers use the DNS settings from the host system.
+You can change the DNS policy and define separate nameservers and search domains.
+See the Docker [DNS services documentation](https://docs.docker.com/config/containers/container-networking/#dns-services) for more details.
+
+
+![LaunchDockerImageNetworkingInterfaceDNSSettings](/images/SCALE/22.02/LaunchDockerImageNetworkingInterfaceDNSSettings.png "Networking Interface and DNS Policy")  
+
+| Setting | Description |
+|---------|-------------|
+| **Configure Add External Interfaces** | Click **Add** to displays a block of interface settings. |
+| **Interface Configuration** | Required. Select an interface from the **Host Interface** dropdown list. |
+| **Host Interface** | Required. Select a host interface on your system from the dropdown list. |
+| **IP Address Management** | Select an option for how to manage the IP address from the **IPAM Type** dropdown list. |
+| **IPAM Type** | Required. Select an option from the dropdown list to specify the type for IPAM. Options are **Use DHCP** or **Use Static IP**. To add a default route, select **Add route** allow you to enter route destination IP /subnet 0.0.0.0/0. Enter the gatway (for example, *192.168.1.1*). After submitting the docker image, navigate to **Installed Applications**, locate the docker image you added, select **Edit** and change the route destination/subnet to equal 0.0.0.0 /0. |
+| **DNS Policy** | Select the option from the dropdown list that specifies the policy. Default behavior is where Pod inherits the name resolution configuration from the node that the pods run on. If **None** is specified, it allows a pod to ignore DNS settings from the Kubernetes environment. Options are:<br>Use Default DNS Policy where Pod inherits the name resolution configuration from the node**.<br>**Kubernetes internal DNS is prioritized and resolved first.** If the domain does not resolve with internal kubernetes DNS, the DNS query forwards to the upstream nameserver inherited from the node. This useful if the workload to access other services, workflows, using kubernetes internal DNS.<br>**For Pods running with hostNetwork and wanting to prioritize internal kubernetes DNS should make use of this policy.**<br>**Ignore DNS settings from the Kubernetes cluster**. |
```

Comment:
This section of the table is not displaying correctly.

Reply:
fixed